### PR TITLE
Changelog for 1.10.0-RC1

### PR DESCRIPTION
**Pull request by 6543**
finaly ;)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,298 @@ This changelog goes through all the changes that have been made in each release
...
+  * cmd/serv: actually exit after fatal errors (#7458)
+  * Fix an issue with some pages throwing 'not defined' js exceptions (#7450)
+  * fix Dropzone.js integration (#7445)
+  * Fix regex for issues in commit messages (#7444)
+  * Diff: Fix indentation on unhighlighted code (#7435)
+  * Only show "New Pull Request" button if repo allows pulls (#7426)
+  * Upgrade macaron/captcha to fix random error problem (#7407)
+  * create class for inline positioned lists (#7393)
+  * Fetch refs for successful testing for tag (#7388)
+  *  add missing template variable on organisation settings  (#7385)
```
Comment (6543) [inline CHANGELOG.md]: format

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,298 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix pull request commit status in user dashboard list (#8321)
+  * Fix repo_admin_change_team_access always checked in org settings (#8319)
+  * Update to github.com/lafriks/xormstore@v1.3.0 (#8317)
+  * Show correct commit status in PR list (#8316)
+  * Bugfix for image compare and minor improvements to image compare (#8289)
+  * Update xorm (#8286)
+  * Fix API for edit and delete release attachment (#8285)
+  * Fix nil object access in some conditions when parsing cross references (#8281)
+  * Fix label count (#8267)
+  * Do only show teams access for organization repositories on collaboration setting page (#8265)
```
Comment (guillep2k) [inline CHANGELOG.md]: This title should need some rework. Perhaps:
```suggestion
  * Only show teams access for organization repositories on collaboration setting page (#8265)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,298 @@ This changelog goes through all the changes that have been made in each release
...
+  * Rewrite reference processing code in preparation for opening/closing from comment references (#8261)
+  * Fix assets key on release webhook (#8253)
+  * Allow registration when button is hidden (#8237)
+  * Fix release API URL generation (#8234)
+  * Fix milestone num_issues (#8221)
+  * MS Teams webhook misses commit messages (#8209)
+  * Fix data race (#8204)
+  * Fix team user api (#8172)
+  * Fix pull merge 500 error caused by git-fetch breaking behaviors (#8161)
+  * [fix #7384] make show private icon when repo avatar set (#8144)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * Make show private icon when repo avatar set (#8144)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,298 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.0](https://github.com/go-gitea/gitea/releases/tag/v1.10.0) - 2019-10-14
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
## [1.10.0-RC1](https://github.com/go-gitea/gitea/releases/tag/v1.10.0-rc1) - 2019-10-14
```

**Review by jolheiser — CHANGES_REQUESTED** [review]
Shouldn't this be an RC first?

File: CHANGELOG.md [hunk 5]
```diff
@@ -122,20 +414,20 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
   * Move add to hook queue for created repo to outside xorm session. (#7682) (#7675)
   * Show protection symbol if needed on default branch (#7660) (#7668)
   * Hide delete/restore button on archived repos (#7660)
-  * Fix bug on migrating milestone from github (#7665) (#7666) 
+  * Fix bug on migrating milestone from github (#7665) (#7666)
```
Comment (lunny) [inline CHANGELOG.md:384]: Why these changes added here?

Comment (6543) [reply CHANGELOG.md:384]: Because of attom

Comment (6543) [reply CHANGELOG.md]: Are this on noicy?

Comment (6543) [reply CHANGELOG.md]: I can try to rebase ...

Comment (6543) [reply CHANGELOG.md:384]: But it only delete useles space at the end so this wasent something bad i thought ... @lunny

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,296 @@ This changelog goes through all the changes that have been made in each release
...
+  * Update golangci to v1.19.1 (#8414)
+  * Fix lfs locks (#8361)
+  * Make encoding tests independent of LOCALE settings (#8018)
+  * Fix local runs of ssh-requiring integration tests (#7855)
+  * integration tests: Use t.Helper() (#7654)
+* TRANSLATION
+  * Latvian translation for home page (#8468)
+  * Add home template italian translation (#8352)
+  * fix misprint (#7452)
+* BUILD
```
Comment (lafriks) [inline CHANGELOG.md:229]: Imho whole build and docs sections can be removed

Comment (6543) [reply CHANGELOG.md:229]: OK

Comment (6543) [reply CHANGELOG.md:229]: @lafriks BUILD and DOCS is also part of changelog for 1.9.0 ?!?

Comment (guillep2k) [reply CHANGELOG.md:229]: I'd leave BUILD only for this entry; it seems relevant:
```
* use go 1.13 (#8088)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,296 @@ This changelog goes through all the changes that have been made in each release
...
+  * Move line number to :before attr to hide from search on browser (#8002)
+  * Changed black color to white for (read) number label on issue list page (#8000)
+  * [Branch View] show "New Pull Request" Button only if posible (#7977)
+  * Fix hook problem in #7754 (#7854)
+  * Prevent Commit Status and Message From Overflowing On Branch Page (#7800)
+  * Fix global search result CSS, misc CSS tweaks (#7789)
+  * Tweak label border CSS (#7739)
+  * Fix create menu item widths (#7708)
+  * Extract the username and password from the mirror url (#7651)
+  * [Branch View] Protection Symbole (delete dublicate) (#7624)
```
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
  * [Branch View] Delete duplicate protection symbol (#7624)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,296 @@ This changelog goes through all the changes that have been made in each release
...
+  * Move all mail related codes from models to services/mailer (#7200)
+  * Support git.PATH entry in app.ini (#6772)
+  * Support setting cookie domain (#6288)
+  * Move migrating repository from frontend to backend (#6200)
+  * Delete releases attachments if release is deleted (#6068)
+* SECURITY
+  * Ignore mentions for users with no access (#8395)
+  * Be more strict with git arguments (#7715)
+  * reserve .well-known username (#7637)
+* TESTING
```
Comment (guillep2k) [inline CHANGELOG.md]: Following @lafriks suggestion, perhaps the TESTING section can also be removed.

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,296 @@ This changelog goes through all the changes that have been made in each release
...
+  * Only show "New Pull Request" button if repo allows pulls (#7426)
+  * Upgrade macaron/captcha to fix random error problem (#7407)
+  * create class for inline positioned lists (#7393)
+  * Fetch refs for successful testing for tag (#7388)
+  * add missing template variable on organisation settings (#7385)
+  * fix post parameter - on issue list - unset assignee (#7380)
+  * fix/define autochecked checkboxes on issue list in firefox (#7320)
+  * only return head: null if source branch was deleted (#6705)
+* ENHANCEMENT
+  * vendor: update mvdan.cc/xurls/v2 to v2.1.0 (#8495)
```
Comment (zeripath) [inline CHANGELOG.md:139]:
```suggestion
  * Add nofollow to sign in links (#8509)
  * vendor: update mvdan.cc/xurls/v2 to v2.1.0 (#8495)
```

**Review by zeripath — APPROVED** [review]
I don't mind the Testing, Build and Docs sections remaining - but equally they could be removed.

File: CHANGELOG.md (hunk 6 above)
Comment (6543) [reply CHANGELOG.md:229]: done - I only leave go1.13

**Review by 6543 — COMMENTED** [review]
I'm ready if you are?

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
dont merge!

File: CHANGELOG.md (hunk 9 above)
Comment (6543) [reply CHANGELOG.md:139]: this was already done by me - but i merged this sugestion ... :(

-> fixed

**Comment by 6543** [comment]
ready :rocket:

**Review by jolheiser — APPROVED** [review]
